dev/date-picker.html

```diff
@@ -4,27 +4,70 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Date picker</title>
+    <title>Date Picker</title>
     <script type="module" src="./common.js"></script>
+
     <script type="module">
-      import '@vaadin/date-picker';
-      import '@vaadin/tooltip';
-      const isDateDisabled = (date) => {
-        // Exclude weekends and the 16th day of each month:
-        const checkDate = new Date(0, 0);
-        checkDate.setFullYear(date.year);
-        checkDate.setMonth(date.month);
-        checkDate.setDate(date.day);
-        return checkDate.getDay() === 0 || checkDate.getDay() === 6 || checkDate.getDate() === 16;
-      }
-      const picker = document.querySelector('vaadin-date-picker');
-      picker.isDateDisabled = isDateDisabled;
-      picker.min = '2023-11-01';
```

Comment:
We should use these features in the "bells & whistles" example, to get the disabled date styles tested.